stix_shifter_modules/elastic_ecs/stix_transmission/api_client.py

```diff
@@ -15,9 +15,14 @@ def __init__(self, connection, configuration):
         headers = dict()
         url_modifier_function = None
         auth = configuration.get('auth')
-        self.indices = configuration.get('elastic_ecs', {}).get('indices', None)
+        indices = connection.get('indices', None)
+
+        if type(indices) == str:
```

Comment:
Could do this in one line as `if indicies and type(indices) == str:`

Reply:
yup. updated